From 34458435857984fe7a176081e7c7f86d5ea5fc3a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 15 Jun 2019 14:47:25 +0200
Subject: [PATCH 1/1] efi_loader: EFI_SIMPLE_TEXT_OUTPUT_PROTOCOL definition

EFI_SIMPLE_TEXT_OUTPUT_PROTOCOL.Reset() is a function and not a void *
pointer.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/efi_api.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/efi_api.h b/include/efi_api.h
index d7d95edd4d..4de5d208f5 100644
--- a/include/efi_api.h
+++ b/include/efi_api.h
@@ -577,7 +577,9 @@ struct simple_text_output_mode {
 #define EFI_ATTR_BG(attr)        (((attr) >> 4) & 0x7)
 
 struct efi_simple_text_output_protocol {
-	void *reset;
+	efi_status_t (EFIAPI *reset)(
+			struct efi_simple_text_output_protocol *this,
+			char extended_verification);
 	efi_status_t (EFIAPI *output_string)(
 			struct efi_simple_text_output_protocol *this,
 			const efi_string_t str);
-- 
2.20.1

